package.json

```diff
@@ -90,8 +94,8 @@
     "morgan": "1.2.0",
     "ms": "0.7.1",
     "node-sass": "3.7.0",
+    "notifications-panel": "1.1.13",
```

Comment:
Yeah, not ours but looks like it got re-ordered. Perhaps someone manually added it in the wrong place?